examples/test/Menu_test.js

```diff
@@ -0,0 +1,158 @@
+
+    await clickOnMenuAggregationItem(t, measureCell, '.s-menu-aggregation-sum');
+    await t.expect(getPivotTableFooterCell(0, 0).textContent).eql(totalValues.sum[0]);
+    await t.expect(getPivotTableFooterCell(0, 3).textContent).eql(totalValues.sum[3]);
+    await t.expect(getPivotTableFooterCell(0, 4).textContent).eql(totalValues.sum[4]);
+
+    await clickOnMenuAggregationItem(t, measureCell, '.s-menu-aggregation-sum');
+    await t.expect(getPivotTableFooterCell(0, 0).exists).eql(false);
+});
+
+test('should turn on/off multiple totals', async (t) => {
+    const measureCell = getMeasureCell(0);
+    const measureGroup = getMeasureGroupCell(0);
+
+
+    await clickOnMenuAggregationItem(t, measureCell, '.s-menu-aggregation-sum');
+    await t.expect(getPivotTableFooterCell(0, 0).textContent).eql(totalValues.sum[0]);
+    await t.expect(getPivotTableFooterCell(0, 3).textContent).eql(totalValues.sum[3]);
+    await t.expect(getPivotTableFooterCell(0, 4).textContent).eql(totalValues.empty);
+    await t.expect(getPivotTableFooterCell(1, 0).exists).eql(false);
+
+    await clickOnMenuAggregationItem(t, measureGroup, '.s-menu-aggregation-max');
+    await t.expect(getPivotTableFooterCell(0, 0).textContent).eql(totalValues.sum[0]);
+    await t.expect(getPivotTableFooterCell(0, 3).textContent).eql(totalValues.sum[3]);
+    await t.expect(getPivotTableFooterCell(0, 4).textContent).eql(totalValues.empty);
+    await t.expect(getPivotTableFooterCell(1, 0).textContent).eql(totalValues.max[0]);
+    await t.expect(getPivotTableFooterCell(1, 3).textContent).eql(totalValues.max[3]);
+    await t.expect(getPivotTableFooterCell(1, 4).textContent).eql(totalValues.max[4]);
+    await t.expect(getPivotTableFooterCell(2, 0).exists).eql(false);
+
+    await clickOnMenuAggregationItem(t, measureGroup, '.s-menu-aggregation-sum');
+    await t.expect(getPivotTableFooterCell(0, 0).textContent).eql(totalValues.sum[0]);
+    await t.expect(getPivotTableFooterCell(0, 3).textContent).eql(totalValues.sum[3]);
+    await t.expect(getPivotTableFooterCell(0, 4).textContent).eql(totalValues.sum[4]);
+    await t.expect(getPivotTableFooterCell(1, 0).textContent).eql(totalValues.max[0]);
+    await t.expect(getPivotTableFooterCell(1, 3).textContent).eql(totalValues.max[3]);
+    await t.expect(getPivotTableFooterCell(1, 4).textContent).eql(totalValues.max[4]);
+
+    await clickOnMenuAggregationItem(t, measureGroup, '.s-menu-aggregation-sum');
+    await t.expect(getPivotTableFooterCell(0, 0).textContent).eql(totalValues.max[0]);
+    await t.expect(getPivotTableFooterCell(0, 3).textContent).eql(totalValues.max[3]);
+    await t.expect(getPivotTableFooterCell(0, 4).textContent).eql(totalValues.max[4]);
+    await t.expect(getPivotTableFooterCell(1, 0).exists).eql(false);
+
+    await clickOnMenuAggregationItem(t, measureGroup, '.s-menu-aggregation-max');
+    await t.expect(getPivotTableFooterCell(0, 0).exists).eql(false);
+});
```

Comment:
Please add test case for

1. Add Sum on group
2. Turn Sum off on one measure
3. Turn Sum off on other measure

The Sum should beturned off on the group as well